dependency_metrics/datadog_utils.py

```diff
@@ -64,7 +63,13 @@ def send_metric(name, value, metric_type, tags=None):
             'tags': [f'{key}:{value}' for key, value in tags.items()]
         }]
     }
-    requests.post(url, headers=headers, json=json.dumps(payload))
+    response = requests.post(url, headers=headers, json=payload)
+    if not response.ok:
+        print(
```

Comment:
Not debugging, just logging errors if encountered. I could probably use the `logging` library instead, but this was the first instance where I logged something more than just the command output in this project, so still just used `print`. Open to switching over to `logging` here

Reply:
I am not sure if we should have logging at all
1. The information in `payload`. Are those just numbers and no project data right?
2. Considering the amount of requests we get, I don't think you will be able to locate that information easily in logs.
3. Is there something that flags that the metric was not sent successfully, that makes someone search for these logs? If there isn't any trigger, then the log loses its relevance.

Reply:
Ah I see your concerns. If it helps, this is in a separate project from hq that will only be run by a daily github action, so if the github action fails (or a manual run fails), it is useful to know why it failed.

Reply:
okay. So no sensitive data gets into this.
I am still not sure how and when this would be useful. But you think it will be needed, fine to add it.

Reply:
So this is a python package that would be installed by a specific project, and used within the context of that project to send metrics to datadog whenever run (either manually or via a dedicated github action). This requires some setup, ensuring api and app keys are exported to the environment prior to making the request to datadog. My main goal with this print statement is to make it obvious why the request to datadog failed to the user of the library, because it is possible the issue is just related to improper setup that the user can resolve. Or worst case, the user can report a bug with a detailed message of why the request failed.

Reply:
Thanks @gherceg 

Just confirming that this is not blocking.